src/main/java/org/opensearch/geospatial/action/upload/geojson/UploadGeoJSONRequest.java

```diff
@@ -14,43 +14,37 @@
 import java.io.IOException;
 import java.util.Objects;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+
 import org.opensearch.action.ActionRequest;
 import org.opensearch.action.ActionRequestValidationException;
 import org.opensearch.common.bytes.BytesReference;
 import org.opensearch.common.io.stream.StreamInput;
 import org.opensearch.common.io.stream.StreamOutput;
 import org.opensearch.rest.RestRequest;
 
+@AllArgsConstructor()
```

Comment:
() not required.

Also you can move the @Getter on the top of the class. If all attributes of the class requires @Getter